tests/test_integration.py

```diff
@@ -40,7 +40,7 @@
     // rx(pi / sin(3 * 4 / 2 - 2)) | [b, c];
 end;
 
-obs o(a):
+obs o(a)[0, 1]:
```

Comment:
Is this necessary? Or did you add this to be pedantic?

Reply:
This is necessary for the test to pass. Since the serialization will always add the declared wires, the integration tests will fail without it. One of the annoyances with these tests.